src/test/resources/mockdata/fundsValidator/validateFundDistributionsRequest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "subTotal": 10,
+  "currency": "USD",
+  "adjustments": [
```

Comment:
you can format json using this tool: https://jsonformatter.curiousconcept.com/#

Reply:
done